.github/integration/tests/sda/70_rotate_key_test.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/sh
+set -e
```

Comment:
```suggestion
set -eo pipefail
```
With this setting, it can also catch errors in pipelines, e.g. `curl | jq`

Reply:
not posix compliant, see [here](https://www.shellcheck.net/wiki/SC3040)